content/en/agent/iot/_index.md

```diff
@@ -0,0 +1,209 @@
+
+## Setup
+
+### Requirements
+
+The IoT Agent is available as DEB and RPM packages for Linux devices running on x64, arm64 (ARMv8), and ARMv7 architectures.
+
+#### Resources
+
+IoT devices are typically more resource constrained than cloud infrastructure hosts. The IoT Agent is built to have a minimal footprint and consume minimal network bandwidth.
+
+Exact resource requirements depend on usage. Datadog found the following when testing the IoT Agent (v7.20) internally:
+
+- CPU: 0.5% (on a VM with 2 Intel Xeon VCPUs)
+- Memory: 36 MB
+- Network bandwidth: 237 bps up / 79 bps down
+- Disk: 63 MB
+
+### Installation
+
+#### Automatic
+
+To automatically download and install the correct IoT Agent for your operating system and chipset architecture, use the following command:
+
+```shell
+DD_API_KEY=<YOUR_DD_API_KEY> DD_SITE="{{< region-param key="dd_site" code="true" >}}" DD_AGENT_MAJOR_VERSION=7 DD_AGENT_FLAVOR=datadog-iot-agent bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+```
+
+#### Manual
+
+{{< tabs >}}
+{{% tab "DEB" %}}
+
+To manually install the IoT Agent on Debian-based operating systems, run the following commands:
+
+1. Update `apt` and install `apt-transport-https` to download through HTTPS:
+    ```bash
+    sudo apt-get update
+    sudo apt-get install apt-transport-https
+    ```
+
+2. Set up the Datadog deb repo on your system and import Datadog's apt key:
+    ```bash
+    sudo sh -c "echo 'deb https://apt.datadoghq.com/ stable 7' > /etc/apt/sources.list.d/datadog.list"
+    sudo apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 A2923DFF56EDA6E76E55E492D3A80E30382E94DE
+    ```
+
+3. Update `apt` and install the IoT Agent:
+    ```bash
+    sudo apt-get update
+    sudo apt-get install datadog-iot-agent
+    ```
+
+{{< site-region region="us" >}}
+
+<ol start="4">
+  <li><p>Copy the example config and plug in your API key:</p>
+  <pre><code>DD_API_KEY=&lt;YOUR_DD_API_KEY&gt; ; sudo sh -c "sed 's/api_key:.*/api_key:$DD_API_KEY/' /etc/datadog-agent/datadog.yaml.example > /etc/datadog-agent/datadog.yaml"</code></pre>
+  </li>
+</ol>
+
+{{< /site-region >}}
+{{< site-region region="eu" >}}
+
+<ol start="4">
+  <li><p>Copy the example config and plug in your API key:</p>
+  <pre><code>DD_API_KEY=&lt;YOUR_DD_API_KEY&gt; ; sudo sh -c "sed 's/api_key:.*/api_key:$DD_API_KEY/' /etc/datadog-agent/datadog.yaml.example > /etc/datadog-agent/datadog.yaml"</code></pre>
+  <li><p>Set your Datadog site:</p>
+  <pre><code>sudo sh -c "sed 's/# site:.*/site: datadoghq.eu/' /etc/datadog-agent/datadog.yaml > /etc/datadog-agent/datadog.yaml.new && mv /etc/datadog-agent/datadog.yaml.new /etc/datadog-agent/datadog.yaml</code></pre>
+  </li>
+  </li>
+</ol>
+
+{{< /site-region >}}
+
+<ol start="6">
```

Comment:
```suggestion
<ol start="5">
```

Reply:
There's 2 bullet points above this one (4 and 5) so this group needs to start with `6`.